src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.html

```diff
@@ -0,0 +1,47 @@
+<ng-container *ngIf="gatewayGroup$ | async as gateways">
+  <cd-table
+    #table
+    [data]="gateways"
+    [columns]="columns"
+    columnMode="flex"
+    selectionType="single"
+    identifier="name"
+    (updateSelection)="updateSelection($event)"
+    (fetchData)="fetchData()">
+  </cd-table>
+</ng-container>
+
+<ng-template #dateTpl
+             let-events="data.value">
+  <span *ngIf="events?.length">{{ events[0].created | date:'EEE d MMM, yyyy' }}</span>
+</ng-template>
+<ng-template #gatewayTpl
```

Comment:
When naming things, let's make it clear what it does.
```suggestion
<ng-template #gatewayStatusTpl
```